### ISSUE TITLE
PWA Theme

### ISSUE THREAD

**Issue by nealey** [dirtbags/moth#68]
Let's make a theme that:

- Defines and provides a `moth-common.js` which provides all the "example puzzle 5" niceities (roll this back into the default theme)
- Provides a broadcast notification system
- Tells teams what points they have scored, with an option for admins to disable this functionality
- Downloads everything associated with a puzzle category, so people can run detached
- Downloads everything associated with itself, so people can run detached and hit "reload" and it all still works

Basically, let's make an actual client that's a progressive web app.

**Comment by nealey** [comment]
This repo is not going to support anything more advanced than a very basic theme.
